core/test/ParaGridIO_input_test.py

```diff
@@ -0,0 +1,86 @@
+import netCDF4
+import numpy as np
+import time
+import calendar
+import math
+from errno import ENXIO
+
+sec_per_hr = 3600
+hr_per_day = 24
+
+if __name__ == "__main__":
+    # Set up the argument parsing
+#    import argparse
+#    parser = argparse.ArgumentParser(description = "Create fake forcing file for the PAraGridIO test")
+#    parser.add_argument("--file", dest="file", required = True, help = "A restart file containing the target grid information.")
+#    parser.add_argument("--start", dest = "start", required = True, help = "The ISO start date for the forcing file.")
+#    parser.add_argument("--stop", dest = "stop", required = True, help = "The ISO end date for the forcing file.")
+#    args = parser.parse_args()
+    # read the date range
+#    start_time = time.strptime(args.start, "%Y-%m-%d")
+#    stop_time = time.strptime(args.stop, "%Y-%m-%d")
+    #(unix_times, era5_times, topaz4_times) = create_times(start_time, stop_time)
+
+    # read a grid spec (from a restart file)
+#    root = netCDF4.Dataset(args.file, "r", format = "NETCDF4")
+#    structgrp = root.groups["structure"]
```

Comment:
Removed the commented code.

Reply:
What a mess `black` makes of my beautiful maps ;_;

Reply:
I have raised the formatting as a separate issue. We don't have to use black or any other tool but it might help with consistency. We can discuss later. (see #405)